src/dnd/Draggable.jsx

```diff
@@ -1,31 +1,27 @@
 import React, { Component } from 'react';
-import { DndContext } from './DndContext';
 
 class Draggable extends Component {
   draggable = null;
 
   mouseDown = setDraggable => () => {
     if (this.draggable) {
-      setDraggable(this.draggable, this.props.id);
+      setDraggable(this.draggable, this.props.ID);
     }
   };
 
   setRef = (el) => { this.draggable = el; };
 
   render() {
     return (
-      <DndContext.Consumer>
-        {context => (
-          <div // eslint-disable-line jsx-a11y/no-static-element-interactions
-            className="drag"
-            onMouseDown={this.mouseDown(context.setDraggable)}
-            ref={this.setRef}
-          >
-            <p>{this.props.id}</p>
-            {this.props.children}
-          </div>
-        )}
-      </DndContext.Consumer>
+
+      <div // eslint-disable-line jsx-a11y/no-static-element-interactions
+        className="draggable"
+        onMouseDown={this.mouseDown(this.props.setDraggable)}
```

Comment:
No sense to pass `this.props.setDraggable`

It's inside props

Restructure `mouseDown`

Reply:
ready